gdb/printing.py

```diff
@@ -249,20 +248,28 @@ def _get_children(self, max_count: int) -> None:
                 # In order to do this we need our error message as a valid C++
                 # expression. We pack it into a raw string literal with a custom
```

Comment:
Please move this comment to the line where `CreateValueFromExpression` is called with the `R"GALA_ERROR(...)GALA_ERROR"` string literal.

Reply:
done - moved the msg